### ISSUE TITLE
Creating thread at once, without publishing first post (Web-interface masotodon.ocial)

### ISSUE THREAD

**Issue by Twilightssuperb** [mastodon/mastodon#27349]
### Pitch

Sorry if it was already asked.

Implement this like at Twitter.

It's inconvenient and annoying to post first item, then reply to it manually, then...

### Motivation

UX economy and time saving.

+1 more feature for users who are migrating from Twitter or thinking about it.

**Comment by BakaDalek** [comment]
Agree, and using this PLEASEEE PLEASE PLEASEEEE make all the replies unlisted by default

This way, threads appear in user timelines with the start post only, not the back to back to back to back replies people do with threads today.

A built in way to do threads could solve this!

**Comment by Erikcht** [comment]
Is this what you are asking for?
![9](https://github.com/user-attachments/assets/97674d79-e42e-43c5-b304-b558ca4e8a76)

If so, it is a function that is needed.

PS: I prefer the UI version made of this on the Threads platform.
![post](https://github.com/user-attachments/assets/1a821bd0-ec49-4cc6-91bf-a6aee81e2845)
